.ci/Jenkinsfile

```
@@ -86,6 +86,24 @@ pipeline {
         }
       }
     }
+    /**
+     Execute integration tests.
+     */
+    stage('TestIntegration') {
```

There are a couple of missing env vars

```suggestion
    stage('TestIntegration') {
            environment {
              HOME = "${env.WORKSPACE}"
              PATH = "${env.HOME}/go/bin:${env.PATH}"
            }
```